storage/range_raftstorage.go

```diff
@@ -348,8 +348,12 @@ func (r *Range) updateRangeInfo() error {
 		return util.Errorf("failed to lookup zone config for Range %s: %s", r, err)
 	}
 	r.SetMaxBytes(zone.RangeMaxBytes)
-	// TODO(kkaneda): Update configHashes and other fields as well (#1362)?
 
+	// Gossip configs and update configHashes if a lease is held. This is normally unnecessary
```

Comment:
This only works in tests because we only have one replica and leases are instantaneous. Does it ever help in any other case? If there is no lease then configHashes will not actually be updated. The intention of #1362 was to copy all the range's internal state, regardless of lease status. I'm not sure that it's actually worthwhile to copy configHashes since they will be updated when/if the node gets the lease (and don't really matter otherwise).